src/fu-util-common.c

```diff
@@ -1552,7 +1552,9 @@ fu_util_remote_to_string (FwupdRemote *remote, guint idt)
 static void
 fu_security_attr_append_str (FwupdSecurityAttr *attr, GString *str)
 {
-	if (fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+	if (fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_OBSOLETED)) {
```

Comment:
I'd argue we probably want to sort this list by flags now when printing it.  Success/Fail/Obsoleted.

Reply:
I pushed this